public/locales/fr/translation.json

```diff
@@ -359,13 +359,17 @@
     "signin": {
       "title": "Exemple d'Application Web Azure"
     },
+    "accessdenied": {
+      "title": "Accès non autorisé",
+      "errormessage": "Les resources n'existent pas ou vous n'avez pas la permission d'y accéder.",
+      "signouttimeout": "Vous allez être déconnecté automatiquement dans {{seconds}}... secondes"
+    },    
     "unauthorized": {
```

Comment:
Can we remove the `unauthorized` block if it is no longer useful ? (or merge it with `accessdenied`)